### PR TITLE
Update comments on AdvancedFluentLayoutExtensions

### PR DESCRIPTION
**Pull request by munkii**
I meant to submit this PR almost a year ago. When I was first looking at AutoLayout and FluentLayout i found the lack of comments made it harder for me to understand so I thought this might help others

### CODE REVIEW

**Comment by gshackles** [comment]
Thanks for this! I'm certainly open to adding some comments here to help clarify things as long as they're useful and not too intrusive, though part of the API's design is also aimed to try to be self-explanatory as well. One concern I'd have is adding it for just some methods, which could potentially make things more confusing, but that also might not be a big deal.

One nitpick I'd have about this change in particular is the language around `create a LayoutConstraint`. I think it would be better to speak in terms of our API since we're trying to abstract away the underlying constraint system for the most part.

**Comment by munkii** [comment]
I get your point about "language" and abstraction. I felt like I wanted to know what was going and so ended up expressing that in the comment. What about "Used to create a constraint", minor change I know but probably enough to demphasize underlying API. 

I wasn't sure about the comments on some of the methods. There were some methods I had used a lot so was more comfortable writing the comments for those and thought some comments is better than none

**Comment by gshackles** [comment]
Yeah, that works

**Comment by munkii** [comment]
Comments updated

File: Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs [hunk 1]
```diff
@@ -17,36 +17,130 @@ public static class AdvancedFluentLayoutExtensions
...
+        /// <param name="previous">The related view</param>
+        /// <param name="margin">Any vertical space to increase the offset by</param>
+        /// <remarks>Uses NSLayoutAttribute.Top, Implies a sibling, vertically stacked layout relationship</remarks>
+        /// <returns>A FluentLayout</returns>
         public static FluentLayout Below(this UIView view, UIView previous, nfloat? margin = null) =>
 			view.Top().EqualTo().BottomOf(previous).Plus(margin.GetValueOrDefault(DefaultMargin));
 
+        /// <summary>
+        /// Used to create a constraint that offsets the View's bottom most edge from the 
+        /// top edge of <paramref name="following"/> a.k.a. the secondItem.
```
Comment (gshackles) [inline Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs]: `following` should be `previous` here

File: QuickLayout.Touch/Views/UpdateConstraintsView.cs [hunk 2]
```diff
@@ -17,46 +17,50 @@ public override void ViewDidLoad()
...
+		    if (RespondsToSelector(new Selector("edgesForExtendedLayout")))
+		    {
+		        EdgesForExtendedLayout = UIRectEdge.None;
+		    }
 
-			var label = new UILabel 
+		    var label = new UILabel 
 			{ 
-				Text = "Update this label's height constraint height constant and active settings",
+				Text = "This label has a Height constraint applied to it. \r\nThe toggle switch is bound to the Contraint's Active boolean and the slider to the Constraint's Constant value.",
```
Comment (gshackles) [inline QuickLayout.Touch/Views/UpdateConstraintsView.cs]: Not a big deal but technically you only need `\n` on iOS. A good rule of thumb is just to use `Environment.NewLine` and then you never need to remember, especially now that C# supports string interpolation 😄

**Comment by gshackles** [comment]
Going to close this out for now but still happy to consider this again in the future! 😄

**Pull request closed without merging**